erdjs/src/transaction.spec.ts

```diff
@@ -1,11 +1,11 @@
-import { assert } from "chai";
-import { Transaction } from "./transaction";
+import {assert} from "chai";
```

Comment:
The whitespace changed from `{ foo }` to `{foo}`. Is this intended?

Reply:
yes, the linter changed that for me after re-formatting the code